src/NServiceBus.MessagingBridge/Hosting/BridgeHostedService.cs

```diff
@@ -18,7 +18,7 @@ public async Task StartAsync(CancellationToken cancellationToken = default)
             .ConfigureAwait(false);
     }
 
-    public Task StopAsync(CancellationToken cancellationToken = default) => runningBridge.Stop(cancellationToken);
+    public Task StopAsync(CancellationToken cancellationToken = default) => runningBridge?.Stop(cancellationToken) ?? Task.CompletedTask;
```

Comment:
I spotted this issue when I saw that we got a null ref here when one of the tests failed. Ie if something throws during start this will prevent a null ref from masking the actual problem